plugins/kotlin-dataframe/testData/selectionDslTestUtils.kt

```diff
@@ -0,0 +1,60 @@
+import org.jetbrains.kotlinx.dataframe.*
+import org.jetbrains.kotlinx.dataframe.annotations.*
+import org.jetbrains.kotlinx.dataframe.api.*
+import org.jetbrains.kotlinx.dataframe.io.*
+import org.jetbrains.kotlinx.dataframe.schema.DataFrameSchema
+
+@DataSchema
+interface Name {
+    val firstName: String
+    val lastName: String
+}
+
+@DataSchema
+interface Person {
+    val name: DataRow<Name>
+    val age: Int
+    val city: String?
+    val weight: Int?
+    val isHappy: Boolean
+}
+
+@DataSchema
+interface FirstNames {
+    val firstName: String
+    val secondName: String?
+    val thirdName: String?
+}
+
+@DataSchema
+interface Name2 {
+    val firstName: DataRow<FirstNames>
+    val lastName: String
+}
+
+@DataSchema
+interface Person2 {
```

Comment:
sometimes I suppose it's bad that some numerical types as Long/Double are not presented in this dataset

Reply:
true, I suppose weight could be a double actually, that's pretty realistic